src/pt/lycantoons/src/eu/kanade/tachiyomi/extension/pt/lycantoons/LycanToonsDto.kt

```diff
@@ -0,0 +1,97 @@
+package eu.kanade.tachiyomi.extension.pt.lycantoons
+
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import keiyoushi.utils.tryParse
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.jsonPrimitive
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.TimeZone
+
+@Serializable
+data class PopularResponse(
+    val data: List<SeriesDto>,
+    val pagination: PaginationDto? = null,
+)
+
+@Serializable
+data class SeriesDto(
+    val title: String,
+    val slug: String,
+    @SerialName("coverUrl")
+    val coverUrl: String? = null,
+    val author: String? = null,
+    val artist: String? = null,
+    val description: String? = null,
+    val genre: List<String>? = null,
+    val status: String? = null,
+    val seriesType: String? = null,
+    val capitulos: List<ChapterDto>? = null,
+)
+
+@Serializable
+data class PaginationDto(
+    val page: Int? = null,
+    val totalPages: Int? = null,
+    val hasNext: Boolean? = null,
+)
+
+@Serializable
+data class SearchRequestBody(
+    val limit: Int,
+    val page: Int,
+    val search: String,
+    val seriesType: String,
+    val status: String,
+    val tags: List<String>,
+)
+
+@Serializable
+data class ChapterDto(
+    val id: Int,
+    val numero: JsonElement,
+    @SerialName("createdAt") val createdAt: String? = null,
+    @SerialName("coverUrl") val coverUrl: String? = null,
+    @SerialName("capaUrl") val capaUrl: String? = null,
+    @SerialName("pageCount") val pageCount: Int? = null,
```

Comment:
Unless I'm missing something, you don't need `@SerialName` if the variable name is the exact same.
```suggestion
    val createdAt: String? = null,
    val coverUrl: String? = null,
    val capaUrl: String? = null,
    val pageCount: Int? = null,
```